spec/controllers/events_controller/create_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::EventsController, type: :controller do
+  let!(:event_attributes) { %w[id name date description status user_id] }
+  let(:organizer) { create(:user, role: :organizer) }
+
+  describe 'POST #create' do
+    let(:valid_params) { build(:event, user_id: organizer.id) }
+
+    context 'when user is organizer' do
+      before do
+        @header = organizer.create_new_auth_token
+        request.headers.merge!(@header)
+      end
+
+      context 'when params is valid' do
+        it 'returns json with params of created event' do
+          post :create, params: { event: valid_params.attributes }
+          expect(json_response.keys).to eq(event_attributes)
+          expect(json_response['name']).to eq(valid_params.name)
+        end
+      end
+
+      context 'when params is invalid' do
+        let(:invalid_params) { build(:event, name: nil, description: 'too short', date: nil, status: nil) }
+
+        it 'returns message errors' do
+          post :create, params: { event: invalid_params.attributes }
+          expect(json_response).to include('Name can\'t be blank')
```

Comment:
you can put all errors into array and iterate over this array. In such case there will be only 1 line with expectation instead of 4